reference/6/Microsoft.PowerShell.Core/About/about_Logging.md

```diff
@@ -0,0 +1,144 @@
+---
+ms.date:  2017-12-04
+schema:  2.0.0
+locale:  en-us
+keywords:  powershell
+title:  about_Logging
+---
+
+# About Logging
+
+## Short Description
+
+PowerShell logs details of PowerShell operations, such as starting
+and stopping the program engine and starting and stopping PowerShell
+providers.  it will also log details about PowerShell commands.
+
+## Long Description
+
+The location of PowerShell logs is dependent on the target platform.
+On Windows, PowerShell logs to the event log, on Linux, PowerShell logs
+to syslog, and on MacOS, the os_log logging system is used.
+
+## Viewing the PowerShell event log on Windows
+
+PowerShell logs can be viewed using the Event Viewer. The event log is
+located in the Application and Services Logs group and is named PowerShellCore.
+The associated ETW provider guid is `{f90714a8-5509-434a-bf6d-b1624c8a19a2}'
+
+### Registering the PowerShell event provider on Windows
+
+Unlike Linux or MacOS, Windows requires the event provider to be registered
+before logged events can appear in the event log.  For PowerShell, this is
+accomplished by running the `RegisterManifest.ps1` from an elevated
+PowerShell prompt.
+
+The script is located in the the install directory and should be run from that location.
+
+### Unregistering the PowerShell event provider on Windows
+
+Registering an event provider places a lock in the binary containing the data
+needed to decode events. To update this binary, the provider must first be
+unregistered to release this lock.
+
+To unregister the PowerShell provider, run the `RegisterManifest.ps1` from an
+elevated PowerShell prompt and specify the `-Unregister` switch.  Once the
+upgrade of PowerShell has completed, run `RegisterManifest.ps1` a second time
+to register the updated ETW provider.
+
+## Viewing PowerShell log output on Linux
+
+PowerShell logs the syslog on Linux and any of the tools commonly used to view
+syslog contents may be used.
+
+The format of the log entries uses the following template:
+
+```
+TIMESTAMP MACHINENAME powershell[PID]: (COMMITID:TID:CID) [EVENTID:TASK.OPCODE.LEVEL] MESSAGE
+```
+
+| Field       | Description
```

Comment:
Although markdown doesn't require it and will render it correctly, you should add whitespace to the table to align the columns when viewed as raw text

Reply:
fixed